packages/manager/modules/pci/src/projects/project/gateways/add/translations/Messages_fr_FR.json

```diff
@@ -36,5 +36,6 @@
   "pci_projects_project_public_gateways_add_modal_add_private_network_error": "Une erreur est survenue lors de la création du réseau privé : {{ message }}",
   "pci_projects_project_public_gateways_add_success": "Une nouvelle Gateway a été créée avec succès.",
   "pci_projects_project_public_gateways_add_error": "Une erreur est survenue lors de la création de la Gateway: {{ message }}",
-  "pci_projects_project_public_gateways_add_creating_wait_message": "Votre Gateway est en cours de création. La configuration de tous les composants peut prendre quelques minutes. Veuillez ne pas quitter ou rafraîchir cette page car cela pourrait avoir un impact sur le processus de création."
+  "pci_projects_project_public_gateways_add_creating_wait_message": "Votre Gateway est en cours de création. La configuration de tous les composants peut prendre quelques minutes. Veuillez ne pas quitter ou rafraîchir cette page car cela pourrait avoir un impact sur le processus de création.",
+  "pci_projects_project_public_gateways_add_region_activate": "Cette région n'est pas encore activée. Vous pouvez l'activer sur l'onglet <a href='{{regionsLink}}'>Quota and localisation</a> puis revenir créer votre Gateway."
```

Comment:
**nitpick:** Avoid HTML in translation files

Reply:
Done Anoop